### ISSUE TITLE
How do I minimally add my own directive prefix?

### ISSUE THREAD

**Issue by thedeeno** [sstephenson/sprockets#430]
I'm working on the [haml_coffee_assets](https://github.com/netzpirat/haml_coffee_assets) gem. It implements a Tilt::Template which is responsible for processing haml_coffee files. I'd like to also leverage sprocket's dependency management, but haml_coffee chokes on the built in directive prefixes. I've located the [header pattern here](https://github.com/sstephenson/sprockets/blob/master/lib/sprockets/directive_processor.rb#L47) but I'm unsure of a clean way to override it without clobbering the extensions other gems may make.

Is there an idiomatic way to add my own directive prefixes?

Basically I need to support a prefix that looks like: `-#= ...`
